### PR TITLE
fix:Alignment of insight text

### PR DESCRIPTION
**Pull request by sanketpathak**

### CODE REVIEW

**Comment by sanketpathak** [comment]
show1


Here is how it looks

**Comment by sanketpathak** [comment]
show2

**Comment by sanketpathak** [comment]
@jyasveer  Done with the changes

**Comment by sanketpathak** [comment]
show3

File: src/app/list-element/list-element.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <div class="list-group-item-header" [ngStyle]="{'background': isOpen === true ? '#def3ff' : '#fff'}">
       <div class="check">
         <div class="list-view-pf-checkbox" *ngIf="fromAddDependency !== 'true'">
-          <input #dependency.name type="checkbox" class="check-dep" [ngStyle]="{'top': fromAddDependency === 'true' ? '29px' : '30px'}"
+          <input #dependency.name type="checkbox" class="check-dep"
```
Comment (jyasveer) [inline src/app/list-element/list-element.component.html]: Reference names should not be containing dots. please check

**Pull request merged**